e2e/cypress/integration/collapsed_reply_threads/crt_settings_spec.js

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
```

Comment:
Please add the following guidelines on top of each new spec files. Then please add comments per step and verification so that non-technical reviewer (QA) can easily follow and compare that the steps are according to the written test criteria in Zephyr.
```
// ***************************************************************
// - [#] indicates a test step (e.g. # Go to a page)
// - [*] indicates an assertion (e.g. * Check the title)
// - Use element ID when selecting an element. Create one if none.
// ***************************************************************
```

Same comment to other spec files.